app/src/main/res/values/styles.xml

```diff
@@ -75,7 +75,7 @@
         <item name="mozacPromptLoginEditTextCursorColor">@color/prompt_login_edit_text_cursor_color_normal_theme</item>
         <item name="readerModeStartGradient">@color/readermode_start_gradient_normal_theme</item>
         <item name="readerModeEndGradient">@color/readermode_end_gradient_normal_theme</item>
-
+        <item name="syncedTabsSeparator">@color/synced_tabs_separator</item>
```

Comment:
You'll need to add 3 dimensions here like the other color attrs. Let's rename `synced_tabs_separator` to something like "synced_tabs_separator_normal_theme" and then you'll also have to add a private_theme color defined in the private theme as well which I can assume we can use the same color as dark theme if it wasn't defined (maybe post new screenshots in private and we can quickly verify with UX as well) 🤔